ogr/abstract.py

```diff
@@ -2062,3 +2062,37 @@ def get_forks(self) -> Sequence["GitProject"]:
             Sequence of forks in user's namespace.
         """
         raise NotImplementedError()
+
+
+class Label(OgrAbstractClass):
+    """
+    Represents labels on PRs and issues.
+    """
+
+    def __init__(self, parent: Any) -> None:
+        self._parent = parent
+
+    @property
+    def name(self) -> str:
+        """Name of the label."""
+        raise NotImplementedError()
+
+
+class IssueLabel(Label):
+    @property
+    def issue(self) -> "Issue":
+        """Issue of issue label."""
+        return self._parent
+
+    def __str__(self) -> str:
+        return "Issue" + super().__str__()
```

Comment:
I think that this in combination with the `__str__` in the forge-specific classes will create a rather weird string representation, but in the end… it's still just a string, so I think it's fine.